src/components/Admin/Admin.tsx

```diff
@@ -169,6 +174,13 @@ const Admin = () => {
                 }
                 roomTitle={deleteRoomTitle}
             />
+            <LogoutConfirmationModal
+                open={showLogoutModal}
+                onClose={() => {
+                    handleClose(setShowLogoutModal);
```

Comment:
Why pass the setter function here as an argument when you already have access to it in the function scope?